src/superface-test.test.ts

```diff
@@ -510,6 +570,27 @@ describe.skip('SuperfaceTest', () => {
         mocked(exists).mockResolvedValueOnce(true);
         mocked(readFileQuiet).mockResolvedValueOnce('[]');
         mocked(matchWildCard).mockReturnValueOnce(false);
+        mocked(prepareSuperface).mockResolvedValue({
```

Comment:
mocking of prepareSuperface can be probably move to some top level function to get rid of repeating it in every case.